packages/core/test/actorLogic.test.ts

```diff
@@ -464,6 +465,44 @@ describe('promise logic (fromPromise)', () => {
     const fn2 = signalListenerList[1];
     expect(fn2).toHaveBeenCalled();
   });
+
+  it('can emit updates', async () => {
+    const p = fromPromise(async ({ update }) => {
+      update({ value: 'starting' });
+      update({ value: 'loading', context: { progress: 0.6 } });
+      update({ value: 'finished' });
+      return 'done data';
+    });
+
+    const stuff: Array<{}> = [];
+
+    const actor = createActor(p);
+    actor.subscribe((s) => {
+      stuff.push(s);
+    });
+    actor.start();
+
+    const res = await toPromise(actor);
+
+    expect(res).toEqual('done data');
+
+    expect(stuff).toEqual(
+      expect.arrayContaining([
+        expect.objectContaining({}),
```

Comment:
it feels like a better "assertion" could be put here, this one looks suspicious - maybe we could use `{ status: 'active' }` or smth like that?

Reply:
then it would make sense to include the status in all of those assertions~